src/helpers/countryDropdown.js

```diff
@@ -1,44 +1,15 @@
-import React from "react";
-import { Form, Select } from "antd";
-import { GlobalOutlined } from "@ant-design/icons";
-import countryList from "./countryList";
-const { Option } = Select;
-
-const CountryDropdown = props => {
-  const children = [];
-
-  for (let i = 0; i < countryList.length; i++) {
-    children.push(
-      <Option key={countryList[i].code} value={countryList[i].name}>
-        {countryList[i].name}
-      </Option>
-    );
-  }
-
-  return (
-    <Form.Item
-      name="org_country"
-      rules={[
-        {
-          required: true,
-          message: "Please select the country"
-        }
-      ]}
-    >
-      <Select
-        style={{ width: "100%" }}
-        placeholder={
-          <div style={{ textAlign: "left" }}>
-            <GlobalOutlined style={{ color: "rgba(0,0,0,.4)" }} /> Country
-          </div>
-        }
-        showSearch={true}
-        defaultValue={props.defaultValue}
-      >
-        {children}
-      </Select>
-    </Form.Item>
-  );
+import React, { useState, useMemo } from "react";
+import Select from "react-select";
+import countryList from "react-select-country-list";
```

Comment:
So you have introduced new dependency, but the functionality is the same as previous. What is the use case of this dependency here then?

Reply:
We are using additional file countryLists.js where we keep storing 200+ country name and code manually. So, I was thinking if we can use some react library then in future if something's changes ,  We dont have to manually update it.
other than this , my code   will make the component more reusable. 
*** Previous Code***
like in /Dashboard/index.js  we keep importing importing countryList.js file and run a for loop to append the countries into select element.
![Screenshot from 2021-06-15 20-13-31](https://user-images.githubusercontent.com/52108435/122073614-2dec5f00-ce16-11eb-834c-d0f992679bd0.png)
If we have to use another country selector component we have to keep importing that. 
so , I used this dependency.

Reply:
No matter what, we have to update the dependencies. Because the changes on the latest package will not reflect on ours if we have a downgrade version. So I think its pretty much like doing the same thing, but quite easy when dependency is considered. And I'm not getting how this component is getting more reusable as this is doing the same thing that was doing previously. Only change is the dependency. And if you look into that package, you will see the same logic in there also(using a loop to render countries from a json file). According to my opinion adding more dependencies make the project vulnerable. It is better to remove unnecessary dependencies. Need ideas from @Niweera and @thisuraseniya also regarding this.

Reply:
Yes, I think what @shehand is saying quite correct. Actually this is an unnecessary risk that we are taking by using a third party dependency, where we can use the same logic here. The component is ours so the total control is with us. I think @thisuraseniya can shed some light on this issue too.

Reply:
yeah, I was thinking ,  like In this previous comment , we can remove those extra code when  we  need to use select  organization feature ( it will run under that npm package). But I got your point and I realized my mistake,   this dependencies might be problemetic in future . So , I think we can close this PR @shehand .

Reply:
Agreed @shehand and @Niweera. Let's keep the original countryList.js and avoid the new dependency. Besides I don't think the country list is something that would change frequently.

Ps - if you want to create a reusable country select menu component using the countryList.js, go for it @rijusougata13